android/build.gradle

```
@@ -12,6 +12,7 @@ if (isNewArchitectureEnabled()) {
 }
 
 android {
+    namespace = "io.sentry.react"
```

Lets first check if the namespace property is available not to break older apps with AGP<4.2.
```suggestion
    // Conditional for compatibility with AGP <4.2.
    if (project.android.hasProperty("namespace")) {
        namespace 'io.sentry.react'
    }
```